mifospay/src/main/res/layout/fragment_home.xml

```diff
@@ -65,7 +65,7 @@
 
             <ImageView
                 android:layout_gravity="center_horizontal"
-                android:id="@+id/imageView"
+                android:id="@+id/shape_bottom_sheet_dialog_grip"
```

Comment:
could be something better

Reply:
Um.. what should I name I the drawable name was that so I thought I would be good if it was named the same

Reply:
Yes, @haran2248 please follow the above pattern